module/importer/src/Infrastructure/Action/AttributeValidatorImportAction.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Importer\Infrastructure\Action;
+
+use Ergonode\Attribute\Domain\Query\AttributeQueryInterface;
+use Ergonode\Attribute\Domain\ValueObject\AttributeCode;
+use Ergonode\Core\Domain\ValueObject\TranslatableString;
+use Ergonode\Importer\Infrastructure\Exception\ImportException;
+use Ergonode\Importer\Infrastructure\Validator\AttributeImportValidator;
+
+class AttributeValidatorImportAction
```

Comment:
IMHO this is not import action and is not validator but some kind of validation  service (maybe a filter)